### DISCUSSION TITLE
Compiling egui-winit with wasm-pack

### DISCUSSION THREAD

**Discussion by gadirom** [emilk/egui#3124] [Q&A]
Hi! I use egui-winit to manage egui input and it works on desktop, but when I try to compile it to WASM with wasm-pack I get messages like this:
```
   Compiling arboard v3.2.0
error[E0433]: failed to resolve: could not find `Clipboard` in `platform`
  --> /Users/gadirom/.cargo/registry/src/github.com-1ecc6299db9ec823/arboard-3.2.0/src/lib.rs:68:38
   |
68 |         Ok(Clipboard { platform: platform::Clipboard::new()? })
   |                                            ^^^^^^^^^ could not find `Clipboard` in `platform`

error[E0433]: failed to resolve: could not find `Clear` in `platform`
   --> /Users/gadirom/.cargo/registry/src/github.com-1ecc6299db9ec823/arboard-3.2.0/src/lib.rs:122:31
    |
122 |         Clear { platform: platform::Clear::new(&mut self.platform) }
    |                                     ^^^^^ could not find `Clear` in `platform`
```
This happens even if I set features of egui-winit to empty.

**Comment by gadirom** [comment]
I've got it to compile by setting `default-features = false`, but judging from the online-demo, it should be possible to compile with all the features enabled. Or does the demo use another crate for winit integration?

**Reply by MilesConn** [reply]
Did you ever figure this out?

**Comment by Davidos533** [comment]
Guys, it's really intersting, how to compile it in **wasm** with **winit**, **egui** and support feature [clipboard](https://docs.rs/crate/egui-winit/latest/features#clipboard)?
Any updates?
